src/main/java/org/opensearch/flowframework/exception/FlowFrameworkException.java

```diff
@@ -60,4 +64,9 @@ public FlowFrameworkException(String message, Throwable cause, RestStatus restSt
     public RestStatus getRestStatus() {
         return restStatus;
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        return builder.startObject().field("error", "Request failed with exception: [" + this.getMessage() + "]").endObject();
```

Comment:
Nit: seems redundant to say your request failed when you're already giving the "error" key. Why not just include the message here?  (No strong opinion.)

Reply:
I can change this, took some inspiration from your [BaseExtensionRestHandler::exceptionalRequest](https://github.com/opensearch-project/opensearch-sdk-java/blame/b6ccc75b988932c37c4452cca5d03583dea7b072/src/main/java/org/opensearch/sdk/rest/BaseExtensionRestHandler.java#L195-L197) :)